docs/documentation/installation.md

```diff
@@ -28,19 +29,184 @@ OfficeLife is a Laravel-based application and requires the following technologie
   * PDO PHP Extension
   * Tokenizer PHP Extension
   * XML PHP Extension
-* MySQL or SQLite,
-* Nginx.
+* MySQL, Mariadb or PostgreSQL
+* Apache or Nginx
+
+### Deployment
+
+- We recommend [Fortrabbit](https://www.fortrabbit.com/) and [Platform.sh](https://platform.sh/) as PaaS (Platform As A Service) hosts, or [Forge](https://forge.laravel.com) and [Ploi](https://ploi.io/) as IaaS platforms provisioner, to deploy an OfficeLife instance.
 
-We recommend [Forge](https://forge.laravel.com) or [Ploi](https://ploi.io/) to provision the servers needed to run OfficeLife. You can also use our official Docker image if you prefer.
+- You can also use our official [Docker image](https://github.com/officelifehq/docker/pkgs/container/officelife-dev) if you prefer.
 
-### Configuration
+- You can also download a bundle asset from our [releases](https://github.com/officelifehq/officelife/releases) containing all dependencies at once. The assets also contains gnupg signature files, signed off with the key `0x213F85B7B1B8F93224FA85B138C2BAD715449E85`. You will find the public key on several [keyserver](http://keyserver.ubuntu.com/pks/lookup?search=0x213F85B7B1B8F93224FA85B138C2BAD715449E85&fingerprint=on&op=index), or [here](pubkey.asc).
+
+### Setup
 
 You should configure your web server's document / web root to be the `public` directory. The `index.php` in this directory serves as the front controller for all HTTP requests entering your application.
 
+Once installed, run this command to setup everything:
+```
+php artisan setup
+```
+
 ### Directory permissions
 
 You may need to configure some permissions. Directories within the `storage` and the `bootstrap/cache` directories should be writable by your web server or OfficeLife will not run.
 
+
+## Configuration
+
+All configuration are set through the `.env` file or within environment variables.
+The `.env.example` file contains a list of common configuration you might want to use. These are the principal ones:
+
+
+### Global config
+
+- `APP_ENV`: set to `production` for a production installation
+- `APP_KEY`: this is provisionned by the `php artisan key:generate` command, or you can manually generate it using this command: `echo -n 'base64:'; openssl rand -base64 32`
+- `APP_DEBUG`: this should be set to `false` for a production installation
```

Comment:
```suggestion
- `APP_DEBUG`: this should be set to `false` for a production installation. If set to `true`, this will display all debug traces when an error occurs, so you should never, ever set it to `true` for production use.
```